docs/releases/4.0.txt

```diff
@@ -28,6 +28,29 @@ The Django 3.2.x series is the last to support Python 3.6 and 3.7.
 What's new in Django 4.0
 ========================
 
+``zoneinfo`` is now the default timezone implementation
+-------------------------------------------------------
+
+* The ``zoneinfo`` module is included in the Python stdlib from Python 3.9. It
+  is now the default timezone implementation used by Django.
+  ``timezone.get_default_timezone()`` and related functions return ``zoneinfo``
+  objects by default.
+
+* ``backports.zoneinfo`` is available for Python 3.8.
+
+  ``tzdata`` is needed FILL_THIS_IN(...roughly, when is the DB not installed on
+  local system... — how will a user know?)
```

Comment:
A user will know because they will hit [ZoneInfoNotFoundError](https://docs.python.org/3/library/zoneinfo.html#zoneinfo.ZoneInfoNotFoundError).

I think we should conditionally add tzdata as a dependency if the OS is Windows. This will solve 99,99% of the problem — the only remaining problem being "writing the proper invocation in setup.py" :-)